mrmustard/utils/settings.py

```diff
@@ -87,63 +75,63 @@ def __init__(self):
         self.AUTOSHAPE_MAX: int = 50
         r"""The max shape for the autoshape. Default is ``50``."""
 
-        self.DRAW_CIRCUIT_PARAMS: bool = True
-        "Whether or not to draw the parameters of a circuit."
+        self.ATOL: float = 1e-8
+        r"""The absolute tolerance when comparing two values or arrays. Default is ``1e-8``."""
+
+        self.BS_FOCK_METHOD: str = "vanilla"  # can be 'vanilla' or 'schwinger'
+        r"""The method for computing a beam splitter in the Fock basis . Default is ``vanilla``."""
 
         self.CIRCUIT_DECIMALS: int = 3
-        "The number of decimal places to display when drawing a circuit."
+        r"""The number of decimal places to display when drawing a circuit. Default is ``3``."""
+
+        self.DEBUG: bool = False  # TODO: remove in MM 1.0
+        r"""Whether or not to print the vector of means and the covariance matrix alongside the html representation of a state. Default is ``False``."""
 
         self.DISCRETIZATION_METHOD: str = "clenshaw"
-        "The method used to discretize the Wigner function. Can be ``clenshaw`` (better, default) or ``iterative`` (worse, faster)."
+        r"""The method used to discretize the Wigner function. Can be ``clenshaw`` (better, default) or ``iterative`` (worse, faster). Default is ``clenshaw``."""
+
+        self.DRAW_CIRCUIT_PARAMS: bool = True
+        r"""Whether or not to draw the parameters of a circuit. Default is ``True``."""
 
         self.EQ_TRANSFORMATION_CUTOFF: int = 3  # TODO: remove in MM 1.0
-        "The cutoff used when comparing two transformations via the Choi–Jamiolkowski isomorphism. Default is 3."
+        r"""The cutoff used when comparing two transformations via the Choi–Jamiolkowski isomorphism. Default is ``3``."""
 
         self.EQ_TRANSFORMATION_RTOL_FOCK: float = 1e-3  # TODO: remove in MM 1.0
-        "The relative tolerance used when comparing two transformations via the Choi–Jamiolkowski isomorphism. Default is 1e-3."
+        r"""The relative tolerance used when comparing two transformations via the Choi–Jamiolkowski isomorphism. Default is ``1e-3``."""
 
         self.EQ_TRANSFORMATION_RTOL_GAUSS: float = 1e-6  # TODO: remove in MM 1.0
-        "The relative tolerance used when comparing two transformations on Gaussian states. Default is 1e-6."
+        r"""The relative tolerance used when comparing two transformations on Gaussian states. Default is ``1e-6``."""
 
         self.HOMODYNE_SQUEEZING: float = 10.0  # TODO: remove in MM 1.0
-        "The value of squeezing for homodyne measurements. Default is 10.0."
+        r"""The value of squeezing for homodyne measurements. Default is ``10.0``."""
 
-        self.PROGRESSBAR: bool = True
-        "Whether or not to display the progress bar when performing training. Default is True."
+        self.MAX_BATCH_SIZE: int = 1000
+        r"""The maximum batch size across all modes. Default is ``1000``."""
```

Comment:
Hey, I ran some tests: with a value of 5 million it didn't overflew 8GB. 5 million, if I am correct, corresponds to the maximum size of a float64 array (i.e. np.size(array) <= 5000000 is True) that would be handled in the memory. That's the spirit.
I got up to 7.34GB, starting from 2.6GB. So we may want to have some security and pass 4 million as default instead (because some more RAM may be used in the background than 2.6GB).
Basically, if one can increase this number, it will run faster. If one can't, it would memory-crash.
```suggestion
        self.MAX_BATCH_SIZE: int = 4000000
        r"""The maximum batch size across all modes. Default is ``4000000``."""
```